neurom/apps/morph_stats.py

```diff
@@ -73,34 +83,94 @@ def _stat_name(feat_name, stat_mode):
     return '%s_%s' % (stat_mode, feat_name)
 
 
+def extract_dataframe(neurons, config):
+    """Extract stats grouped by neurite type from neurons.
+
+    Arguments:
+        neurons: a neuron, population or neurite tree
+        config (dict): configuration dict. The keys are:
+            - neurite_type: a list of neurite types for which features are extracted
+              If not provided, all neurite_type will be used
+            - neurite: a dictionary {neurite_feature: mode} where:
+                - neurite_feature is a string from NEURITEFEATURES
+                - mode is an aggregation operation provided as a string such as:
+                  ['min', 'max', 'median', 'mean', 'std', 'raw', 'total']
+
+    Returns:
+        The extracted statistics
+
+    Note:
+        An example config can be found at:
```

Comment:
Just had an idea. What if we use placeholder here for this config example? Because currently it does not look obvious that we adjust this docstring later. With placeholder it would be clearer.

`An example config can be found at: {config_path}`

Later we would use `extract_stats.__doc__ = extract_stats.__doc__.format({'config_path': EXAMPLE_CONFIG})`